content/articles/build-react-calendar-library/index.md

```diff
@@ -0,0 +1,362 @@
+# Build a Calendar in React Using React-Calendar Library
+
+![hero](/engineering-education/build-react-calendar-library/hero.jpg)
```

Comment:
Kindly follow the article formats, I can see you have published quite a number, kindly check on the intro part!